src/ZODB/blob.py

```diff
@@ -942,8 +942,10 @@ def remove_committed_dir(path):
                 filename = os.path.join(dirpath, filename)
                 remove_committed(filename)
         shutil.rmtree(path)
-
-    link_or_copy = shutil.copy
+    try:
+        link_or_copy = shutil.copy
+    except AttributeError:    #pragma: no cover
+        link_or_copy = shutil.copy
```

Comment:
First, this cannot be the correct spot for the `try: .. except: ...` bit: it is inside the `if sys.platform == "win32":` block.  The correct location should be below (line 950).

Second, the two parts of the `try: ... except: ...` test must not have the same content.  Line 950 below should be changed to:

```python
    try:
        link_or_copy = os.link
    except AttributeError:  #pragma: no cover
        link_or_copy = shutil.copy
```

Reply:
Ooops.  You are correct. The changes were meant for line 950